SETUP.md

```diff
@@ -1,44 +1,80 @@
+
+A comprehensive list of permission scopes is available at [api.slack.com/scopes](https://api.slack.com/scopes).
+
+*   Scroll up to the top of the OAuth & Permissions page and click **Install App to Workspace**.
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/550938ed-48f4-4296-9d73-0a3431f30182)
+    You’ll be led through Slack’s OAuth UI, where you should allow your app to be installed to your development workspace.
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/be6977a7-ec87-4d42-b0fc-bcf09d04474c)
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/6fc1643c-53a8-4b02-b3a7-bb84f2dfc046)
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/3eab21ff-2dbc-4640-81de-245123ac265a)
+
+*   Once you authorize the installation, you will be led to  **OAuth & Permissions** page and see a **Bot User OAuth Access Token**.
+    ![](https://slack.dev/bolt-js/assets/bot-token.png)
+
 > Treat your token like a password and keep it safe. Your app uses it to post and retrieve information from Slack workspaces.
 
 # Setting up your project
-- Navigate to your project folder
-```
+
+*   Navigate to your project folder
+
+<!---->
+
 $ cd chaoss-slack-bot
-``` 
-- Copy your Signing Secret from the Basic Information page and then store it in a new environment variable. 
-- Copy your bot (xoxb) token from the OAuth & Permissions page and store it in another environment variable.
+
+*   Copy your Signing Secret from the Basic Information page and then store it in a new environment variable.
+*   Copy your bot (xoxb) token from the OAuth & Permissions page and store it in another environment variable.
 
 **For Linux and macOS:**
-```
-$ export SLACK_SIGNING_SECRET=<your-signing-secret>
-```
-```
-$ export SLACK_BOT_TOKEN=xoxb-<your-bot-token>
-```
+
+    $ export SLACK_SIGNING_SECRET=<your-signing-secret>
+
+<!---->
+
+    $ export SLACK_BOT_TOKEN=xoxb-<your-bot-token>
 
 **For Windows:**
 
-Create a new file in the root of your project, and name it ```.env```. Copy the content of the  [.env.example](https://github.com/chaoss/chaoss-slack-bot/blob/main/.env.example) file and paste into your .env file. Insert your tokens into the quotes. Make sure you include the .env file in your [```.gitignore```](https://www.delftstack.com/howto/git/add-file-to-gitignore/) file, to avoid exposing your secret keys.
+*   Create a new .env file in the root of your project.
+*   Copy the content(Signing Secret) of the  [.env.example](https://github.com/chaoss/chaoss-slack-bot/blob/main/.env.example) file and paste in the new .env file you created.
```

Comment:
I think the (Signing Secret) needs to go

Reply:
Resolved